packages/frameworks-astro/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "include": ["./src/**/*"],
+  "compilerOptions": {
+    "module": "nodenext"
+  },
+  "exclude": ["*.d.ts"]
+}
```

Comment:
```suggestion
{
  "extends": "utils/tsconfig.json",
  "compilerOptions": {
    "lib": ["dom", "dom.iterable", "esnext"],
    "jsx": "react-jsx",
    "outDir": ".",
    "rootDir": "src"
  },
  "include": ["src/**/*"],
  "exclude": ["*.js", "*.d.ts"]
}
```

Reply:
We won't compile to js files, so do we need a comprehensive config? We also don't need `jsx` resolution within this package. Updated to extend utils & use the lib, but happy to discuss adding anything else from this.